http/preview.go

```diff
@@ -79,6 +80,18 @@ func handleImagePreview(w http.ResponseWriter, r *http.Request, file *files.File
 	}
 	defer fd.Close()
 
+	if format == imaging.GIF && size == sizeBig {
+		g, gerr := gif.DecodeAll(fd)
```

Comment:
It doesn't make sense do decode an image. Just send it as is

Reply:
I was stuck in the image encoding thinking 😅 I've replaced it with the rawFileHandler and I pass the gif now directly to the output.

I also moved the logic above the imgProcessor initialization since it's useless to initialize and then not using it.